roles/openshift_facts/library/openshift_facts.py

```diff
@@ -717,7 +717,8 @@ def set_etcd_facts_if_unset(facts):
         try:
             # Add a fake section for parsing:
             ini_str = '[root]\n' + open('/etc/etcd/etcd.conf', 'r').read()
-            ini_fp = StringIO.StringIO(ini_str)
+            # pylint: disable=not-callable
+            ini_fp = StringIO(ini_str)
```

Comment:
It probably needs to change. I had originally thought that the extended tests would cover testing but our CI clusters have integrated etcd.